src/compiler/diagnosticMessages.json

```diff
@@ -1447,6 +1447,10 @@
         "category": "Error",
         "code": 2501
     },
+    "Cannot create an instance of the abstract class '{1}'": {

```

Comment:
Why 1, and not 0 (for the substitution slot)?

I think I like the "Cannot create an instance" message better.